wcivf/assets/scss/main.scss

```diff
@@ -150,3 +150,21 @@ summary {
   cursor: pointer;
   color: $dc-bright-pink;
 }
+
+.ds-scope { 
+  .ds-candidate {
+    background-color: white;
```

Comment:
Can we remove these line for cards and just make the whole background white for the whole site please? This is the direction we're heading in anyway, so may as well do now

Reply:
And keep the card backgrounds grey?

Reply:
The card backgrounds are just transparent - so will end up white when the background is